static/css/marble.css

```diff
@@ -325,17 +330,50 @@ h6{
     margin-bottom: 16px;
 }
 
-.menu-link-white{
+.menu-logo{
+    width: 50px;
+    height: 50px;
+}
+
+.menu-text-white{
     color: var(--text-on-dark);
     text-decoration: none;
-    margin-left: 80px;
-    margin-top: 16px;
-    margin-bottom: 16px;
 }
 
-.menu-logo{
-    width: 50px;
-    height: 50px;
+/*Override Bootstrap classes*/
+.navbar-toggler,
+.navbar-toggler:focus,
+.navbar-toggler:active,
+.navbar-toggler-icon:focus {
+    outline: none;
+    border: none;
+    box-shadow: none;
+}
+
+.mobile-toggler-icon-white.navbar-toggler-icon{
+    background-image: url(
+        "data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 30 30'%3e%3cpath stroke='rgba(255, 255, 255, 1)' stroke-linecap='round' stroke-miterlimit='10' stroke-width='2' d='M4 7h22M4 15h22M4 23h22'/%3e%3c/svg%3e"
```

Comment:
same question ... do we ever need this to be something other than white? The other code seems to imply that we do but there's no current example of the mobile menu text any other colour.

Reply:
I named it white because this icon is from bootstrap and the default colour is dark grey or black. I'm just overriding their icon.

Reply:
Not unless Shruti needs it to be another colour.  That flexibility would be nice, but if it's fine now there's no need to change it.

Reply:
Ah I see, there's another class for the dark colour. Never mind. Sorry about that.